lib/Zureg/Views.hs

```diff
@@ -130,6 +131,15 @@ ticket r@Registrant {..} = template
 
         registrantInfo r
 
+        case rInfo of
+            Nothing                -> mempty
+            Just RegisterInfo {..} -> case riRegisteredAt of 
+                Nothing            -> mempty
+                Just rRegisteredAt -> when(rRegisteredAt >= Time.UTCTime (Time.fromGregorian 2019 5 7) (15*60*60)) $
```

Comment:
I would move this time to some top-level value (eventually we'll put it in the config)